### PR TITLE
SOLR-16755: bin/solr's '-noprompt' option no longer works for examples

### PR DESCRIPTION
**Pull request by janhoy**
https://issues.apache.org/jira/browse/SOLR-16755

The new test fails with `# ERROR: No line found` without this patch.

### CODE REVIEW

**Comment by HoustonPutman** [comment]
@janhoy used bash arrays instead of the string.

We should also do this with `SOLR_SSL_OPTS` as it's the only other env to use `+="...."`, but we can tackle that in a separate issue that isn't backported to 9.2.1

**Comment by janhoy** [comment]
Thanks Houston. I removed the now obsolete shellcheck disable comment. Will commit and backport shortly

File: solr/packaging/test/test_example_noprompt.bats [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+teardown() {
+  # save a snapshot of SOLR_HOME for failed tests
+  save_home_on_failure
+
+  solr stop -all >/dev/null 2>&1
+}
+
+@test "SOLR16755 test works with noprompt" {
+  echo starting solr
```
Comment (epugh) [inline solr/packaging/test/test_example_noprompt.bats:32]: We don't normally echo things out in our bats tests....

**Review by epugh — CHANGES_REQUESTED** [review]
LGTM once two echos stripped out ;-)

**Pull request merged**